03/app.js

```diff
@@ -1,14 +1,30 @@
 console.log('DOM');
 
 const buttonSettings = {
-    attr: {
-        className: 'btn',
-        title: 'super button'
-    },
-    css: {
-        border: '1px solid #336699',
-        padding: '5px 20px',
-        color: '#444'
-    },
-    text: 'Click me!',
-}
\ No newline at end of file
+	attr: {
+		className: 'btn',
+		title: 'super button',
+	},
+	css: {
+		border: '1px solid #336699',
+		padding: '5px 20px',
+		color: '#444',
+		cursor: 'pointer',
+	},
+	text: 'Click me!',
+};
+
+const parentForBtn = document.querySelector(`.parent-for-button`);
+const button = document.createElement('button');
+
+button.classList.add(buttonSettings.attr.className);
+button.textContent = buttonSettings.text;
+button.setAttribute('title', buttonSettings.attr.title);
```

Comment:
Co jeśli w `attr` będzie więcej elementów? Nie wygodniej zrobić to od razu przy pomocy pętli?